benches/bnf.rs

```diff
@@ -0,0 +1,21 @@
+use bnf::Grammar;
+use criterion::{criterion_group, criterion_main, Criterion};
+
+fn examples(c: &mut Criterion) {
+    c.bench_function("parse postal", |b| {
+        let input = std::include_str!("../tests/fixtures/postal_address.terminated.input.bnf");
```

Comment:
importing from the integration tests makes this a little brittle, but it seemed preferable to duplication